### DISCUSSION TITLE
你好，反馈一下易支付问题，不知道能不能收到

### DISCUSSION THREAD

**Discussion by oohu123** [Baiyuetribe/kamiFaka#128] [Bug反馈]
使用易支付的时候，显示调取二维码失败然后支付不了，易支付不应该是转跳支付的吗，本来就没有调取二维码的功能啊，测试了好几款网上的易支付系统，都对接不了，请问怎么解决。

**Comment by Baiyuetribe** [comment]
易支付已终止服务
